application/lib/viewmodel/sign_up_screen_view_model.dart

```diff
@@ -1,13 +1,98 @@
+import 'package:application/model/user_dto.dart';
 import 'package:flutter/material.dart';
+import '../service/database_service.dart';
 import '../utils/route_constants.dart';
+import '../utils/text_strings.dart';
 
 class SignUpViewModel with ChangeNotifier {
-  void NavigateToSpecificHomeScreen(
+  final UserDTO _user = UserDTO(
+      name: '', username: '', email: '', password: '', isOrganizer: false);
+  final AuthenticationService service = AuthenticationService();
+  List<String> errorMessages = [];
+
+  UserDTO get name => _user;
```

Comment:
A name getter valójában a _user.name adattagot kéne visszaadja. Ugyan ez vonatkozik a többi mezőre is. Vagy legyen minden adattagnak gettere, vagy csak a user objektumnak legyen gettere a és ui-on hivatkozzunk annak az adattagjára.